robottelo/ui/locators/menu.py

```diff
@@ -4,6 +4,16 @@
 from selenium.webdriver.common.by import By
 from .model import LocatorDict
 
+# This is the old menu container xpath, was failing in upstream
```

Comment:
@oshtaier ok!

How do you suggest I call it? [foreman|sat] 
(also there is "upstream=true" in settings)